packages/context/src/__tests__/acceptance/bind-decorator.acceptance.ts

```diff
@@ -35,6 +35,12 @@ describe('@bind - customize classes with binding attributes', () => {
   })
   class MyController {}
 
+  @bind({
+    scope: BindingScope.SINGLETON,
+    tags: ['controller', {name: 'my-singleton-controller', type: 'controller'}],
+  })
+  class MySingletonController {}
```

Comment:
What is the minimal `@bind()` configuration needed to create a singleton controller? Would the following work?

```ts
@bind({scope: BindingScope.SINGLETON})
class MySingletonController {}
```

If it is intended to work, could you please add a test that will add such controller class via `app.controller` API and verify that name, type, namespace, etc. was set correctly?